app/controllers/home_page_controller.rb

```diff
@@ -1,4 +1,4 @@
-class HomePageController < ActionController::Base
+class HomePageController < ApplicationController
     def index
 
```

Comment:
this method has empty body, should be an one-liner

Reply:
now it is one-liner!